# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/extensions/test/browser/browser.ini
# Commit: d995d3771ad5
# Full Hash: d995d3771ad549f75d9f53da7bd62a203ae95ac8
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-04-30 08:26:21
# Regressor Bug: 1594572
# File Overlap Count: 1
# Description:
#   Bug 1594572: Schedule worker deletion when primary runnable fails. r=asuth
#   
#   If we try to start a worker too close to shutdown, the main runnable for its
#   thread runs after shutdown has begun, bails out early trying to create a
#   BackgroundChild for its thread. Unlike bail-outs in WorkerPrivate::DoRunLoop,
# ==============================================================================

diff -r 244a33bcd5e7 -r d995d3771ad5 browser/components/extensions/test/browser/browser.ini
--- a/browser/components/extensions/test/browser/browser.ini	Wed Apr 29 16:40:55 2020 +0000
+++ b/browser/components/extensions/test/browser/browser.ini	Thu Apr 30 03:52:56 2020 +0000
@@ -4,10 +4,6 @@
     # We don't want to reset this at the end of the test, so that we don't have
     # to spawn a new extension child process for each test unit.
     dom.ipc.keepProcessesAlive.extension=1
-    # With Fission enabled, this directory spawns many child processes, which
-    # somehow results in frequent shutdown hangs. Reuse the content processes to
-    # avoid this.
-    dom.ipc.keepProcessesAlive.webIsolated.perOrigin=1
     dom.animations-api.core.enabled=true
     dom.animations-api.timelines.enabled=true
     plugin.load_flash_only=false